.nvmrc

```diff
@@ -0,0 +1 @@
+v5
```

Comment:
Don't think this should be committed, probably should be ignored in your local .gitignore.